_posts/2021-10-25-waku-v1-vs-waku-v2.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+name:  "Waku v1 vs Waku v2: bandwidth comparison"
+title:  "Waku v1 vs Waku v2: bandwidth comparison"
+date:   2021-10-25 17:00:00 +0200
```

Comment:
My bad for late review but, move this date forward :D

Reply:
Done in 21749a4 and changed to `11-02`, tomorrow :)